dj_database_url.py

```diff
@@ -112,11 +112,7 @@ def parse(
 
     # Split query strings from path.
     path = spliturl.path[1:]
-    if "?" in path and not spliturl.query:
-        path, raw_query = path.split("?", 2)
-    else:
-        path, raw_query = path, spliturl.query
-    query = urlparse.parse_qs(raw_query)
+    query = urlparse.parse_qs(spliturl.query)
```

Comment:
This came in in https://github.com/jazzband/dj-database-url/pull/59 but seems to work fine without it. Not sure if there was an issue in earlier (pre-3.x) Python parsing or something.